app/src/marketplace/modules/deprecated/purchaseDialog/actions.js

```diff
@@ -184,7 +157,7 @@ export const approvePurchase = () => (dispatch: Function, getState: () => StoreS
             () => {
                 // Start the purchase transaction, we catch the RECEIVE_PURCHASE_HASH action from purchase
                 // in the reducer and proceed to next step.
-                dispatch(buyProduct(product.id || '', subscriptionTimeInSeconds))
+                dispatch(buyProduct(product.id || '', subscriptionTimeInSeconds, 'DATA', 'DONTCARE', 'DONTCARE'))
```

Comment:
The last two parameters are not needed?

Reply:
This is the deprecated purchase dialog, so these extra params (`ethPrice` and `daiPrice`) aren't used but are expected. 
It's a good point though - my assumption is that the new version of the `purchaseDialog` will be in use before Uniswap lands. I.e. community products comes before uniswap. I hope that assumption is somewhat reasonable, otherwise I'll need to retrofit the the new currency selector, etc into the deprecated dialogs 😅